.changeset/ttfi-ttfme-measurement-lies.md

```
@@ -0,0 +1,9 @@
+---
+'@mastra/factory': patch
+---
+
+Fix TTFI/TTFME measurement lies on `source_control_sessions`.
```

markdownlint-cli2 (0.23.2)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)